README.md

```diff
@@ -114,6 +118,23 @@ The app can then be accessed at `localhost:5001`.
 
 Changes made to the source code will not affect the production application server.
 
+## Running the container on Azure
+
+We can run the To-do app on Azure (at tedwhi-todo-app.azurewebsites.net) by deploying the container on an Azure App Service.
+
+First we must deploy the container image to Docker Hub, this can all be done from the CLI:
+```bash
+docker login
+docker build --target production --tag tedwhi/todo-app:prod .
+docker push tedwhi/todo-app:prod .
+```
+
+You should be able to see the image on Docker Hub. (https://hub.docker.com/r/tedwhi/todo-app)
+
+Then, we need to tell our Azure App Service to pull that latest container image and run it. We can do this by sending a `POST` request to our Azure App Service's Webhook.
+
+The Webhook URL can be found in the `Deployment Center` (sic) of the Azure App Service (`tedwhi-todo-app` in `cohort32-33_TedWhi_ProjectExercise` resource group). Simply copy this URL and send a `POST` request to it. The response will contain the URL of an Azure Log Stream where you can track the progress of updating the App Service.
```

Comment:
Thanks for the clear docs, I'm also a fan of explicitly including the application URL (<https://tedwhi-todo-app.azurewebsites.net/>) - it may sound silly but I've definitely joined teams in the past and then struggled to work out where the dev & prod environments were even deployed!